### PR TITLE
Mount 1 disk at a time per VM

### LINKED ISSUE #32: Attach 1 disk at a time

### PR DESCRIPTION
**Pull request by tanaypf9**
Closes #32

### CODE REVIEW

File: v2v-helper/migrate/migrate.go [hunk 1]
```diff
@@ -61,26 +62,69 @@ func (migobj *Migrate) AddVolumestoHost(vminfo vm.VMInfo) (vm.VMInfo, error) {
...
+		log.Printf("Volume %s detached from VM\n", vmdisk.Name)
 	}
+	time.Sleep(1 * time.Second)
+	return nil
+}
 
-	// Get the Path of the attached volume
-	for idx, vmdisk := range vminfo.VMDisks {
-		devicePath, err := openstackops.FindDevice(vmdisk.OpenstackVol.ID)
+func (migobj *Migrate) DeleteAllDisks(vminfo vm.VMInfo) error {
```
Comment (jeremymv2) [inline v2v-helper/migrate/migrate.go:118]: For the sake of consistency, all function calls for operations on Volumes should probably be named accordingly. I think this should be: `DeleteAllVolumes`.

File: v2v-helper/migrate/migrate.go [hunk 2]
```diff
@@ -259,16 +316,20 @@ func (migobj *Migrate) LiveReplicateDisks(vminfo vm.VMInfo) (vm.VMInfo, error) {
 
 func (migobj *Migrate) ConvertDisks(vminfo vm.VMInfo) error {
```
Comment (jeremymv2) [inline v2v-helper/migrate/migrate.go:317]: For the sake of consistency, all function calls for operations on Volumes should probably be named accordingly. I think this should be: `ConvertVolumes`.

Comment (tanaypf9) [reply v2v-helper/migrate/migrate.go]: Fixed

**Review by jeremymv2 — APPROVED** [review]
Great addition!

**Pull request merged**